webapp/utils/text_formatting.jsx

```diff
@@ -425,3 +427,10 @@ export function handleClick(e) {
         browserHistory.push(linkAttribute.value);
     }
 }
+
+//replace all "/" inside <a> tags to "/<wbr />"
+function insertLongLinkWbr(test) {
```

Comment:
I like the idea of adding the line breaks on slashes, but do you mind moving this to [doFormatText just before the call to replaceTokens](https://github.com/mattermost/platform/blob/master/webapp/utils/text_formatting.jsx#L80)? If you do it there, you shouldn't need to worry about any of the special handling to make sure you're not replacing inside of an html tag